omod/src/main/webapp/template/footerFull.jsp

```diff
@@ -45,6 +45,8 @@
 			
 			<span id="codeVersion"><openmrs:message code="footer.version"/>: ${openmrsVersion}</span>
 			
+			<span id="dockerImage"><openmrs:message code="legacyui.footer.extradata"/>: ${openmrsVersion}</span>
```

Comment:
In the end you'll have something like this:
```html
<span id="extraData">${footerExtraData}</span>
```
However this span should be remain hidden if `${footerExtraData}` is blank or empty.